presto-main/src/main/java/com/facebook/presto/operator/window/WindowAnnotationsParser.java

```diff
@@ -37,12 +38,15 @@ private WindowAnnotationsParser() {}
     {
         WindowFunctionSignature[] signatures = clazz.getAnnotationsByType(WindowFunctionSignature.class);
         checkArgument(signatures.length > 0, "Class is not annotated with @WindowFunctionSignature: %s", clazz.getName());
+        WindowFunctionOptions[] options = clazz.getAnnotationsByType(WindowFunctionOptions.class);
```

Comment:
Isn't it better to use `getAnnotationsByType` and assert that we expect only one annotation here, like I suggested before?
By simply using `getAnnotations` here instead, we don't get rid of the problem of possibly having multiple instances of the same annotation. Instead, we're dealing with a problem of having annotations of other types too.